### PR TITLE
fix(biome): run all enabled biome fixers

### PR DESCRIPTION
**Pull request by redbmk**
- based on biome config, will format, lint, and/or sort imports
- adds variable to apply unsafe fixes, disabled by default

fixes: #4754

### CODE REVIEW

**Comment by redbmk** [comment]
@w0rp I have a few other fixes queued up for biome, I just need to make sure I have tests for them and everything. Should I make it all part of one PR or keep them more atomic with separate PRs?

**Comment by Spixmaster** [comment]
@redbmk I highly doubt that waiting this long for a response is worth it. Maybe you can just handle it how you like.

I appreciate your work!

**Comment by hsanson** [comment]
Smaller self-contained PRs are easier to review given limited time maintainers have. Also easier to revert in case something breaks.

File: autoload/ale/fixers/biome.vim
Comment (hsanson) [inline autoload/ale/fixers/biome.vim:7]: No need to check both `1` and `v:true`.  Simply using  `if  (l:unsafe)` should suffice the check.

File: autoload/ale/fixers/biome.vim
Comment (hsanson) [inline autoload/ale/fixers/biome.vim:13]: Since we want to run only the formater, maybe is good idea to also add `--linter-enabled=false` to the command so linter is not executed?

Comment (Spixmaster) [reply autoload/ale/fixers/biome.vim:13]: From what I understand, everything is intended that the user specificed in his configuration file. Both formatting and linting can be set there.

Comment (redbmk) [reply autoload/ale/fixers/biome.vim:13]: Yeah I would prefer to run the fixer for both the linter and the formatter at the same time. WIthout this change it will only run the formatter.

One could add `--linter-enabled=false` to the options if they only wanted to fix the formatting, but there's a bug in `g:ale_biome_options`. It will apply both to the fixer and the LSP, but they take pretty different arguments and biome will bail if you give invalid ones. I think those should be split out into `ale_biome_lsp_options` and `ale_biome_fixer_options` or something

Comment (hsanson) [reply autoload/ale/fixers/biome.vim:13]: I see, agree that is better to have different options for fixer and linter. Can be handled on a separate PR.

**Comment by hsanson** [comment]
There is a linter issue that causes the test to fail.  After that is fixed I can merge this:

```
./autoload/ale/fixers/biome.vim:7 Blank line required before `if`.
```

**Comment by redbmk** [comment]
> There is a linter issue that causes the test to fail. After that is fixed I can merge this:
> 
> ```
> ./autoload/ale/fixers/biome.vim:7 Blank line required before `if`.
> ```

@hsanson Cool I just pushed up a fix that included your oneliner suggestion. That's a lot cleaner, thanks! I checked that linters are passing locally now

**Comment by redbmk** [comment]
Not sure if it would be an issue in the test runners but I noticed if I run the linter and then fixer tests I get an error. I think if it runs fixers first then linters, it wouldn't show up, but the fix I just pushed lets it work either way.

**Comment by Spixmaster** [comment]
What it the issue with the commits. Why do not the tests finish? I would like to inspect the output but can only call the one of "continuous-integration/appveyor/pr".

**Review by hsanson — DISMISSED** [review]
LGTM, thanks

**Comment by hsanson** [comment]
@redbmk if you could fix the conflict I can merge this.

**Comment by redbmk** [comment]
> LGTM, thanks

looks like I need to fix merge conflicts with the other one now. I'll try to do that asap

**Comment by redbmk** [comment]
@hsanson should be good now

**Review by hsanson — APPROVED** [review]
Thanks

**Pull request merged**
